docs/repository-management.md

```diff
@@ -0,0 +1,134 @@
+
+**Milestones**
+
+We put issues into [milestones](https://github.com/wordpress/gutenberg/milestones) to better categorize them. Here are some you might see:
+
+- The next 2 releases we have milestones for (e.g. 2.2, 2.3).
+- [Feature Complete](https://github.com/WordPress/gutenberg/milestone/8): This includes big features and is what will be managing the vision of Gutenberg. All of this would be done before even merge proposal is thought about. Examples here include nesting, drag and drop and extensibility API.
+- [Merge Proposal](https://github.com/WordPress/gutenberg/milestone/22): This will grow over time and include for now most likely bugs and API issues.
+- [Bonus Features](https://github.com/WordPress/gutenberg/milestone/32): Again likely not part of triage and includes nice to haves for the project, if time before merge. A few examples include collaborative editing and footnotes.
+- [Future](https://github.com/WordPress/gutenberg/milestone/35): this is something that is confirmed by everyone as a good thing but doesn’t fall into other criteria.
+
+**Triaging Issues**
+
+To keep the issue backlog healthy, it needs to be triaged regularly. *Triage* is the practice of reviewing existing issues to make sure they’re relevant, actionable, and have all the information they need.
+
+Anyone can help triage the backlog, although you’ll need contributor permission on the Gutenberg repository to modify an issue’s labels or edit its title.
+
+Here are a couple places you can start:
+
+- [All Gutenberg issues without an assigned label](https://github.com/wordpress/gutenberg/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-asc+no%3Alabel)
+- [The least recently updated Gutenberg issues](https://github.com/WordPress/gutenberg/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-asc)
+
+When reviewing the issue backlog, here are some steps you can perform:
+
+- If it’s a bug report, test to confirm the report.
+- If the issue is missing labels, add some to better categorize it.
+- If the issue is duplicate of another already in the backlog, close the issue by commenting with “Duplicate of #<original-id>”. Add any relevant new details to the existing issue.
+- If there was conversation on the issue but no actionable steps identified, follow up with the participants to see what’s actionable.
+- If the title doesn’t communicate the issue, edit it for clarity.
+- If you feel comfortable triaging the issue further, then you can also:
+  - Check that the bug report is valid by debugging it to see if you can track down the technical specifics.
+  - Check if the issue is missing some detail and see if you can fill in those details. For instance, if a bug report is missing visual detail, it’s helpful to reproduce the issue locally and upload a GIF.
+
+
+## Pull Requests
+
+Gutenberg follows a feature branch pull request workflow for all code and documentation changes. At a high-level, the process looks like this:
+
+
+1. Check out a new feature branch locally.
```

Comment:
Can have some complexity for forked repositories (need to add as a remote and fetch).